src/tattoo.ts

```diff
@@ -79,6 +80,26 @@ const cli = cliArgs([
         "Set to clone all repos from remote instead of updating local copies."
   },
   {
+    name: "wctflags",
+    type: String,
+    defaultValue: "-b chrome",
+    description: "Set to specify flags passed to wct."
+  },
+  {
+    name: "released",
+    type: Boolean,
+    defaultValue: false,
+    description:
+        "Set to update repos to the latest release when possible."
+  },
+  {
+    name: "branchconfig",
```

Comment:
I would name this tattoo_config.js or something, so it can be extended with other settings over time